Fix design

KimK456 opened the pull request.

KimK456 committed:
design logo

KimK456 committed:
added home link on logo

danacohen111 commented:
The headline looks nice, maybe add a black framing to make it look bolder 
What about creating general buttons, headlines, colors ...

danacohen111 reviewed (changes requested):
The headline looks nice, maybe add a black framing to make it look bolder
What about creating general buttons, headlines, colors ...

DavidS252 reviewed (changes requested):
in the home page, i think that we need to improve the background behind of the text. it's barely visible, what do you think?

morielnoam committed:
Solved merge conflicts,
organized uniform design for all the project,
added background in the generic design file

morielnoam committed:
moved the content container of the profile pages right after the brand heading without unnecessary margin

KimK456 committed:
fixed signup page

morielnoam committed:
Merge branch 'dev' of github.com:netanelzucaim/SpotwiseFrontend into fix-design

morielnoam committed:
solved the scrolling issue and organized it in every page

morielnoam committed:
fixed bugs of scrolling in map page

Ilana-Belokon2 commented:
looks great!!!!!
the only thing is that all the pages are scrollable. i think they shouldnt be, only it needed

morielnoam committed:
tried to maximize height of the page so the pages will be unscrollable for every screen resolution and size

morielnoam committed:
tried another functionality to maximize height of the page so the pages will be unscrollable for every screen resolution and size

morielnoam committed:
returned the change of the overflow
